lib/Client.js

```diff
@@ -1676,18 +1676,6 @@ class Client extends EventEmitter {
         return this.requestHandler.request("PATCH", Endpoints.GUILD_TEMPLATE_GUILD(guildID, code), true, options).then((template) => new GuildTemplate(template, this));
     }
 
-    /**
-    * Modify a guild's vanity code
-    * @arg {String} guildID The ID of the guild
-    * @arg {String?} code The new vanity code
-    * @returns {Promise<Object>}
-    */
-    editGuildVanity(guildID, code) {
-        return this.requestHandler.request("PATCH", Endpoints.GUILD_VANITY_URL(guildID), true, {
```

Comment:
remove the endpoint too?

Reply:
`getGuildVanity` should still work
https://github.com/abalabahaha/eris/blob/e43418dae558a8fc7108fbfd820ff6cd8c6a2db3/lib/Client.js#L2614-L2621

Reply:
*does